cli/create.go

```diff
@@ -85,6 +89,10 @@ func runCreate(args []string) (exit int) {
 		return exitError("couldn't create swarm: --image must be provided")
 	}
 
+	if flags.ImageURI == awsEuWest1CoreOS && flags.UseIgnition {
+		return exitError(fmt.Sprintf("couldn't create swarm: --use-ignition requires a recent CoreOS AMI than '%s'", awsEuWest1CoreOS))
```

Comment:
ignition doesn't work with our default CoreOS version cause it is old, you need to use newer ones